heterogeneity/Documentation_JV_Summer2022

```diff
@@ -0,0 +1,194 @@
+Summary of the work perform during the Summer of 2022 by Jorge Valdebenito on the Labs project. 
+
+This project revisit the work of Antonio Ramos Rivera and looks to replicate his draft for the California labs project. 
+The goal is to replicate his work and extend it to the entire country. Unfortunately, given the lack of documentation this was not possible during the summer. 
+We were only able to partially replicate the results for California. 
+Most of the work consisted on running the python scripts in the correct order and troubleshoot the possible mistakes and missing information we found on the pythion scripts. 
+Most of the work was made in Python using the Spyder app. 
+
+The topics I worked are the following:
+1)	Creating a working environment in Python 
```

Comment:
Your numbering is hard to follow because you use "1), 2),..." for the highest-level breakdown of tasks, but then you re-use it within "1)". You can just change to something different, like Roman numerals, a), b),... or whatever else looks good to you,